### ISSUE TITLE
[AZURE] Add missing metrics using CLI

### ISSUE THREAD

**Issue by geostant** [elastic/beats#25171]
**Describe the enhancement:**
Similar to https://github.com/elastic/beats/issues/25065 there is a need to ingest metrics for azure
using the azure CLI.

As a cloud-infra team we would like to separate our ways from the cloud provider's monitoring and alerting in-house tools and start using metricbeats and stack for monitoring and alerting.

While researching what is available in the modules I found missing metrics that I would like to be added via the CLI.

**Describe a specific use case for the enhancement or feature:**

- Azure Autoscaling max limit about to be reached 
  calculation when `(capacity/maxCapacity)*100 >= 90%`
  - CLI: profiles[].capacity.maximum (the upper limit of max instances in a group)
  - CLI: sku.capacity (in service count of instances)

- Azure Autoscaling failed to spin new capacity
  calculation when `capacity > desired capacity after 15 minutes` (they should match once capacity is up and running) 
  - CLI: profiles[].capacity.default (the count of instances that we desire to run)
  - CLI: sku.capacity

So to sum up, I'm missing for AZURE CLI the following
- sku.capacity
- profiles[].capacity.maximum
- profiles[].capacity.default

To grab with the CLI the instance_group data you can run the following:
`az monitor autoscale list --subscription $SUBSCRIPTION_NAME -g $RESOURCE_GROUP_NAME`
`az vmss list --subscription $SUBSCRIPTION_NAME -g $RESOURCE_GROUP_NAME`

Rel: https://github.com/elastic/cloud/issues/75184
